lib/ui/painting/image_generator.cc

```diff
@@ -133,7 +148,39 @@ bool BuiltinSkiaCodecImageGenerator::GetPixels(
   if (prior_frame.has_value()) {
     options.fPriorFrame = prior_frame.value();
   }
-  return codec_generator_->getPixels(info, pixels, row_bytes, &options);
+  SkEncodedOrigin origin = codec_->getOrigin();
+
+  SkPixmap dst(info, pixels, row_bytes);
+  SkPixmap tmp;
+  SkBitmap tmpBitmap;
```

Comment:
nit: naming - flutter uses snake_case for variable names, and prefers to spell things out (although dst/tmp are pretty commonly used).

Reply:
Done.